.github/workflows/nightly.yml

```diff
@@ -0,0 +1,34 @@
+name: nightly build
+
+on:
+  # To trigger this workflow manually, you can use the following curl command:
+  # curl -XPOST -u "USERNAME:PERSONAL_TOKEN" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" https://api.github.com/repos/awslabs/djl-demo/dispatches --data '{"event_type": "nightly-build"}'
+
+  # Make sure you create your personal token with repo access. Follow steps in
+  # https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line
+  # to create your personal token.
+  repository_dispatch:
+    types: [nightly-build]
+  schedule:
+    - cron:  '0 10 * * *'
+
+jobs:
+  documentation:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```

Comment:
Do you need java here? we don't have any notebook to run here

Reply:
We have different java integrations in there right ? Do we maybe need anything in there right now or probably need in the future ?

Reply:
As long as we don't have notebook to run, we don't need java to build the site

Reply:
Ok. I'll make changes to it then.

Reply:
Let's remove it as unnecessary, all we need is python to convert md files to html. We will probably keep notebook in main repo.